lib/whistle/program.ex

```diff
@@ -267,18 +269,18 @@ defmodule Whistle.Program do
       |> Plug.Conn.put_resp_content_type("text/html")
       |> Plug.Conn.send_resp(200, resp)
     else
+      {:view, _} ->
```

Comment:
This is really weird. I don't understand why I had to swap place with the match to get dyalixir happy.